pkg/didcomm/protocol/didexchange/service.go

```diff
@@ -39,11 +61,22 @@ type provider interface {
 type Service struct {
 	outboundTransport transport.OutboundTransport
 	store             storage.Store
+	ch                chan didCommChMessage
+	event             []chan<- event.DIDCommEvent
 }
 
 // New return didexchange service
 func New(store storage.Store, prov provider) *Service {
-	return &Service{outboundTransport: prov.OutboundTransport(), store: store}
+	svc := &Service{
+		outboundTransport: prov.OutboundTransport(),
```

Comment:
There should be a TODO that we actually need an outbound dispatcher rather than the transport directly.

The dispatcher needs to call Pack prior to using the outbound transport.

Reply:
[Added](https://github.com/hyperledger/aries-framework-go/compare/6ca38f34c66c2017dee287d5ecd2734ca169e7b7..5839b0f509b0932a67abd562fb6a05dc181f2e08#diff-f761c91e753eba1c42a68076df06ce12R73) a TODO. Issue #259.